store/presenters/presenters.go

```diff
@@ -28,44 +28,51 @@ func LogListeningAddress(address common.Address) string {
 }
 
 // ShowEthBalance returns the current Eth Balance for current Account
-func ShowEthBalance(store *store.Store) (string, error) {
+func ShowEthBalance(store *store.Store) (map[string]interface{}, error) {
 	if !store.KeyStore.HasAccounts() {
 		logger.Panic("KeyStore must have an account in order to show balance")
 	}
 	account, err := store.KeyStore.GetAccount()
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	address := account.Address
 	balance, err := store.TxManager.GetEthBalance(address)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
-	result := fmt.Sprintf("ETH Balance for %v: %v", address.Hex(), balance)
 	if balance.IsZero() {
-		return result, errors.New("0 Balance. Chainlink node not fully functional, please deposit ETH into your address: " + address.Hex())
+		return nil, errors.New("0 Balance. Chainlink node not fully functional, please deposit ETH into your address: " + address.Hex())
```

Comment:
Optional nitpick:

For all of these, I would return an empty `map[string]interface{}{}` rather than nil. And for this one in particular, could we return `keysAndValues, balanceIsZeroErr`?

Would this allow you to get rid of the `if/else` in the calling function?

Reply:
I'll sort this, I thought it was doing the same on master with outputting `<nil>` but it isn't. I'll make sure it behaves the same.

Reply:
Just changed it so it always returns the kv. Behaviour is same as before now with 0 balance:
```
2018-10-09T17:27:44Z [WARN]  0 Balance. Chainlink node not fully functional, please deposit ETH into your address: 0xb8bb735142a407A2279b3499AFFc8489812A678C cmd/local_client.go:98           
2018-10-09T17:27:44Z [INFO]  ETH Balance for 0xb8bb735142a407A2279b3499AFFc8489812A678C: 0.000000000000000000 cmd/local_client.go:99           address=0xb8bb735142a407A2279b3499AFFc8489812A678C balance=0.000000000000000000 
2018-10-09T17:27:44Z [INFO]  Link Balance for 0xb8bb735142a407A2279b3499AFFc8489812A678C: 0.000000000000000000 cmd/local_client.go:102          address=0xb8bb735142a407A2279b3499AFFc8489812A678C balance=0.000000000000000000
```